snap/local/runtime-helpers/bin/go-services-wrapper.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+# wait for consul to come up
+$SNAP/bin/wait-for-consul.sh $1
+
+# if we get here, just assume that consul is up, otherwise this 
+# service will fail after 10 seconds
```

Comment:
Is 10s the default timeout for all the core services?

Reply:
yes it's hard-coded in the go code for the edgex-go services